routes/web.php

```diff
@@ -52,3 +53,5 @@
 //フォロー/フォロワー一覧表示
 Route::get('/users/{user}/following',[App\Http\Controllers\FollowsController::class, 'followings'])->name('followinglist');
 Route::get('/users/{user}/follower',[App\Http\Controllers\FollowsController::class, 'followers'])->name('followerlist');
+
+Route::get('/notification', [App\Http\Controllers\NotificationController::class,'index']);
```

Comment:
以前はたしかエラーが出てしまった…とありましたが
今のNanaさんの知識であれば対応出来る気がするので、再度ここを下記に修正し、エラー解決をしてみましょう
```suggestion
Route::get('/notifications', [App\Http\Controllers\NotificationController::class,'index']);
```